jaxrs/src/main/java/org/jboss/as/jaxrs/JaxrsTransformers.java

```diff
@@ -45,12 +56,29 @@ public String getSubsystemName() {
 
     @Override
     public void registerTransformers(SubsystemTransformerRegistration subsystemRegistration) {
-        registerTransformers_2_0_0(subsystemRegistration);
+        final ModelVersion[] modelVersions;
+        final ModelVersion target;
+        if (isByPassTransformers()) {
+            target = MODEL_VERSION_1_0_0;
+            modelVersions = new ModelVersion[] {MODEL_VERSION_1_0_0};
+        } else {
+            target = MODEL_VERSION_2_0_0;
+            modelVersions = new ModelVersion[] {MODEL_VERSION_2_0_0, MODEL_VERSION_1_0_0};
+        }
+        ChainedTransformationDescriptionBuilder chainedBuilder =
+                TransformationDescriptionBuilder.Factory.createChainedSubystemInstance(subsystemRegistration.getCurrentSubsystemVersion());
+
+        // Differences between 3.0.0 and 2.0.0
+        ResourceTransformationDescriptionBuilder builder300 =
+                chainedBuilder.createBuilder(subsystemRegistration.getCurrentSubsystemVersion(), target);
+        registerTransformers_3_0_0(builder300);
+
+        // Create chained transformers.
+        chainedBuilder.buildAndRegister(subsystemRegistration, modelVersions);
```

Comment:
@kabir All of the above LGTM but can you confirm?

Reply:
@bstansberry @jamezp Although I don't totally get the isBypassTransformers stuff I think it looks fine. 
It seems we only care about transforming one version delta (CURRENT to 1.0.0; or CURRENT to 2.0.0) and we account for that in the builder.
Then the registration process seems to take that into account. If tests pass both with and without org.wildfly.jaxrs.unsupported.bypass.2.0.0.transformer it should be ok

Reply:
@kabir The test actually fails if the property is set, but that should be expected. The reason for the property is to allow WildFly 19, 20 and 21 servers to **not** reject those attributes in a mixed domain. I doubt it will be used much TBH, but it seemed like we should have a way to bypass it just in case.